end2end/server/mock_aikido_core.py

```diff
@@ -93,6 +111,17 @@ def mock_reset():
     events = [] # Reset events
     return jsonify({})
 
+@app.route('/mock/set_protection', methods=['POST'])
+def mock_set_protection():
+    req = request.get_json()
+    global responses
+    responses["config"]["endpoints"][1]["forceProtectionOff"] = bool(req["api_pets_create"])
+    responses["config"]["endpoints"][2]["forceProtectionOff"] = bool(req["api"])
+    responses["config"]["configUpdatedAt"] = int(time.time()*1000)
+    responses["configUpdatedAt"] = int(time.time()*1000)
+
+    return jsonify({})
```

Comment:
Seems very specific, any way we can make this more generic?

Reply:
Would require a 2nd change to e2e tests, that the config is easier to be updated, I agree but just not for this PR right now